db/migrate/20180424141617_azure_backslash_to_forward_slash.rb

```diff
@@ -0,0 +1,75 @@
+class AzureBackslashToForwardSlash < ActiveRecord::Migration[5.0]
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStack < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class OrchestrationStackOutput < ActiveRecord::Base
+    belongs_to :stack, :class_name => 'OrchestrationStack'
```

Comment:
class_name should be fully qualified with the migration namespac.  e.g. AzureBackslashToForwardSlash::OrchestrationStack